shared/packages/worker/src/worker/workers/genericWorker/expectationHandlers/quantelClipCopy.ts

```diff
@@ -338,6 +339,7 @@ async function lookupCopyTargets(
 	return lookupAccessorHandles<QuantelMetadata>(
 		worker,
 		exp.endRequirement.targets,
+		exp.id,
```

Comment:
Would it make sense to pass around a context object, instead of a bare ID? Since this needs to be added to a lot of methods, and the pattern of "too many method arguments" has already been identified - making this be an object would make it much more future-proof.